docs/contributors/contributors.md

```diff
@@ -0,0 +1,39 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Authors and Contributors
+
+## Authors
+
+Names are displayed in alphabetical order, using the author's last name. 
+
+### Tegveer Ghura
+
+![](Tegveer.jpg)
+
+Hello :wave:  I am Tegveer Ghura and I recently completed an enthralling and rewarding summer internship with Ploomber as a Data Scientist. I am currently a graduate student in the MS in Data Science & Analytics program at Georgetown University and am working towards establishing myself in the industry after graduating in May 2024. I also hold a Bachelor’s degree in Economics from Boston University and have a year’s worth of experience in Finance.
+
+The internship with Ploomber has kickstarted my career in Tech and honed my Data Engineering skills to a whole new extent. Moreover, I wrote production-quality code using the JupySQL package while documenting its functionality, which provided me a deep, intuitive level of understanding the dynamics of the JupySQL package with everyday Data Science practices. I also enjoyed teamwork at Ploomber while building the SQL course and learned a host of new skills, including ETL pipelining and code containerization, from my peers. I highly recommend applying to Ploomber to those looking to gain an enriching internship experience as well as using its packages to streamline project workflows!
+
```

Comment:
Change Tegveer's bio to:

Tegveer recently completed an enthralling and rewarding summer internship with Ploomber as a Data Scientist. He is a graduate student in the MS in Data Science & Analytics program at Georgetown University and is working towards establishing himself in the industry after graduating in May 2024. He also holds a Bachelor’s degree in Economics from Boston University and has a year’s worth of experience in Finance.

The internship with Ploomber has kickstarted his career in Tech and honed his Data Engineering skills to a whole new extent. He wrote production-quality code using the JupySQL package while documenting its functionality, which provided him a deep, intuitive level of understanding the dynamics of the JupySQL package with everyday Data Science practices. He also enjoyed teamwork at Ploomber while building the SQL course and learned a host of new skills, including ETL pipelining and code containerization, from his peers. He highly recommends applying to Ploomber to those looking to gain an enriching internship experience as well as using Ploomber's packages to streamline project workflows!

Reply:
added your bio